src/test/java/com/microsoft/sqlserver/jdbc/SQLServerConnectionTest.java

```diff
@@ -449,42 +449,6 @@ public void testConnectionPoolGetTwice() throws SQLException {
         }
     }
 
-    /**
-     * Tests whether connectRetryCount and connectRetryInterval are properly respected in the login loop. As well, tests
-     * that connection is retried the proper number of times.
-     */
-    @Test
-    public void testConnectCountInLoginAndCorrectRetryCount() {

```

Comment:
why was this test removed what does it have to do with what changed in this PR??